### ISSUE TITLE
Type error in ModifiedSince

### ISSUE THREAD

**Issue by bezin** [alexdebril/rss-atom-bundle#208]
Hi there, 

first of all thank you for providing this bundle! I am trying to implement it into Contao CMS, but ran into some issues:

```
Debril\RssAtomBundle\Request\ModifiedSince::getModifiedSince(): Argument #1 ($request) must be of type Symfony\Component\HttpFoundation\Request, null given, called in [redacted]/vendor/debril/rss-atom-bundle/src/Request/ModifiedSince.php on line 22
```

This happens on cache warming, because than [$this->requestStack->getCurrentRequest()](https://github.com/alexdebril/rss-atom-bundle/blob/master/src/Request/ModifiedSince.php#L22) always returns null. This also happens in any cli command I tried. 

I use the latest version of the bundle with Symfony 5.4

**Comment by alexdebril** [comment]
thanks! it's released in v5.1.3

**Comment by bezin** [comment]
Thanks a lot for reacting so quickly 🚀
